index.js

```diff
@@ -2,7 +2,15 @@ module.exports = function getIpfs (opts) {
   opts = opts || {}
 
   return new Promise(function (resolve, reject) {
-    if (window.ipfs) return resolve(window.ipfs)
+    if (window.ipfs) {
+      // forward-compatible migration
+      // https://github.com/ipfs-shipyard/ipfs-companion/issues/589
+      if (typeof window.ipfs.enable === 'function') {
+        return resolve(window.ipfs.enable())
```

Comment:
Pass wanted permissions...so we don't have to do another release??

```suggestion
        return resolve(window.ipfs.enable(opts.permissions))
```